src/main/java/seedu/savenus/model/food/Food.java

```diff
@@ -147,13 +141,10 @@ public boolean equals(Object other) {
 
         Food otherFood = (Food) other;
         return otherFood.getName().equals(getName())
-                && otherFood.getPrice().equals(getPrice())
-                && otherFood.getDescription().equals(getDescription())
-                && otherFood.getTags().equals(getTags())
-                && otherFood.getCategory().equals(getCategory())
-                && otherFood.getLocation().equals(getLocation())
-                && otherFood.getOpeningHours().equals(getOpeningHours())
-                && otherFood.getRestrictions().equals(getRestrictions());
+                && (otherFood.getDescription().equals(getDescription())
+                        && otherFood.getLocation().equals(getLocation())
+                        && otherFood.getRestrictions().equals(getRestrictions())
+                        && otherFood.getOpeningHours().equals(getOpeningHours()));
```

Comment:
Remember to include into your Design Considerations in the Developer Guide why you dictate these to deem the foods as equals.